lib/vector-config-macros/src/ast/field.rs

```diff
@@ -23,6 +23,7 @@ impl<'a> Field<'a> {
         is_virtual_newtype: bool,
     ) -> darling::Result<Field<'a>> {
         let original = serde.original;
+
```

Comment:
Do you have a good rule-of-thumb for line breaking (assuming this was you and not auto-fmting). I agree it looks better, but often feel odd about adding breaks whenever I feel like it.

Reply:
Hmm, my rule-of-thumb is a bit subjective but: for functions, I typically group variables that are common/needed for the rest of the function body at the top, and then try to use line breaks to group up individual steps:

- alright, convert this input to the intended format
- alright, now extract the data we need
- alright, for all the data we extracted, transform it
- alright, now re-package the transformed data and return it
- etc